feature_engine/outliers/base_outlier.py

```diff
@@ -149,9 +149,7 @@ class WinsorizerBase(BaseOutlier):
 
     If `capping_method='mad'` fold is the value to multiply the MAD.
 
-    If `capping_method='quantiles'`, fold is the percentile on each tail that should
-    be censored. For example, if fold=0.05, the limits will be the 5th and 95th
-    percentiles. If fold=0.1, the limits will be the 10th and 90th percentiles.
+    If `capping_method='quantiles'`, check unit_test
```

Comment:
The documentation needs to be very specific and let the user know anything they need. We can't send them to check unit_test.

Could you please rephrase it so that it reflects the current functionality?

Reply:
Sorry I added that line to check if the unit_test bug is fixed.
I forgot to change it back.